framework/tlvf/yaml/tlvf/WSC/WSC_Attributes.yaml

```diff
@@ -0,0 +1,397 @@
+    _type: eWscAttributes
+    _value: ATTR_MANUFACTURER
+  data_length:
+    _type: uint16_t
+    _value: WSC_MAX_NAME_LENGTH
+  data:
+    _type: char
+    _length: [ WSC_MAX_NAME_LENGTH ]
+
+sWscAttrModelName:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_MODEL_NAME
+  data_length:
+    _type: uint16_t
+    _value: WSC_MAX_NAME_LENGTH
+  data:
+    _type: char
+    _length: [ WSC_MAX_NAME_LENGTH ]
+
+sWscAttrModelNumber:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_MODEL_NUMBER
+  data_length:
+    _type: uint16_t
+    _value: WSC_MAX_NAME_LENGTH
+  data:
+    _type: char
+    _length: [ WSC_MAX_NAME_LENGTH ]
+
+sWscAttrSerialNumber:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_SERIAL_NUMBER
+  data_length:
+    _type: uint16_t
+    _value: WSC_MAX_NAME_LENGTH
+  data:
+    _type: char
+    _length: [ WSC_MAX_NAME_LENGTH ]
+
+sWscAttrPrimaryDeviceType:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_PRIMARY_DEV_TYPE
+  data_length:
+    _type: uint16_t
+    _value: WSC_PRIMARY_DEV_TYPE_LENGTH
+  category_id:
+    _type: uint16_t 
+  oui:
+    _type: uint8_t
+    _length: [ WSC_PRIMARY_DEV_TYPE_OUI_LENGTH ]
+  sub_category_id:
+    _type: uint16_t 
+
+sWscAttrDeviceName:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_DEV_NAME
+  data_length:
+    _type: uint16_t
+    _value: WSC_MAX_NAME_LENGTH
+  data:
+    _type: char
+    _length: [ WSC_MAX_NAME_LENGTH ]
+
+sWscAttrRfBands:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_RF_BANDS
+  data_length:
+    _type: uint16_t
+    _value: 1
+  data:
+    _type: eWscRfBands
+
+sWscAttrAssociationState:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_ASSOC_STATE
+  data_length:
+    _type: uint16_t
+    _value: 1
+  data:
+    _type: eWscAssoc
+    _value: WSC_ASSOC_NOT_ASSOC
+
+sWscAttrDevicePasswordID:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_DEV_PASSWORD_ID
+  data_length:
+    _type: uint16_t
+    _value: 2
+  data:
+    _type: eWscValues16
+    _value: DEV_PW_PUSHBUTTON
+
+sWscAttrConfigurationError:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_CONFIG_ERROR
+  data_length:
+    _type: uint16_t
+    _value: 2
+  data:
+    _type: eWscValues16
+    _value: WSC_CFG_NO_ERROR
+
+sWscAttrOsVersion:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_OS_VERSION
+  data_length:
+    _type: uint16_t
+    _value: WSC_OS_VERSION_LENGTH
+  data:
+    _type: uint32_t
+    _value: 0x80000001
+
+sWscAttrVersionExtension:
+  _type: struct
+  attribute_type:
+    _type: eWscAttributes
+    _value: ATTR_VENDOR_EXTENSION
+  data_length:
+    _type: uint16_t
+    _value: WSC_VENDOR_EXTENSIONS_LENGTH
+  data:
+    _type: uint8_t
+    _length: [ WSC_VENDOR_EXTENSIONS_LENGTH ]
```

Comment:
Vendor extension is a tricky one...

For sure, you need the oui as a separate field. So you could have:
```
cWscAttrVersionExtension:
  _type: class
  type:
    _type: eWscAttributes
    _vlaue: ATTR_VENDOR_EXTENSION
  length: uint16_t
  oui:
    _type: eWscVendorExtension
    _length: 3
    _value_const: WSC_VENDOR_EXTENSION_WFA
  ie_version2: sWscWfaIeVersion2

sWscWfaIeVersion2:
  _type: struct
  ie_type:
    _type: eWscWfaIe
    _value_const: WSC_WFA_IE_VERSION2
  ie_length:
    _type: uint8_t
    _value_const: 1
  ie_version2:
    _type: uint8_t
    _value_const: 0
```

Ideally, I would like to have a dynamic list of IEs here, and define the IE as yet another class. But I don't think tlvf really supports that, does it? @tomereli ?

Note that we'll also need to define a vendor extension type for the M2 that combines the Version2 IE with the Multi-AP IE. For the latter, I'm a little concerned about the parsing. In general, it is possible that the Version2 IE and the Multi-AP IE are combined in a single vendor extension attribute, or that it is passed as two separate attributes. Also, the order of the two is not specified, and there may be additional IEs. Does the parser support all that?